action.yml

```diff
@@ -246,6 +246,7 @@ runs:
         WP_CONFIG_PATH="${WP_TESTS_DIR}/wp-tests-config.php"
 
         echo "define( 'IS_UNIT_TEST', true );" >> "${WP_CONFIG_PATH}"
+        echo "define( 'VIP_GO_APP_ENVIRONMENT', 'staging' );" >> "${WP_CONFIG_PATH}"
```

Comment:
Need to define the constant as [object-cache.php](https://github.com/Automattic/vip-go-mu-plugins-built/blob/5cc989e8c388f64a145fda4841f53d84b0182a5e/drop-ins/object-cache.php) needs it defined.